tests/alembic/conftest.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from invenio_db import db as db_
+
+
+from inspirehep.factory import create_app
+from sqlalchemy_utils.functions import create_database, database_exists
+
+
+@pytest.fixture(scope='module')
+def app():
+    """Flask application.
+
+    Creates a Flask application with a simple testing configuration,
+    then creates an application context and inside of it recreates
+    all databases and indices from the fixtures. Finally it yields,
```

Comment:
This `app` does not seem to recreate databases and indices?

Reply:
yep leftover. I am changing it.